src/main/java/trackr/model/menu/ItemProfit.java

```diff
@@ -0,0 +1,54 @@
+package trackr.model.menu;
+
+import static java.util.Objects.requireNonNull;
+import static trackr.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents an Item's profit in the Item list.
+ * Guarantees: immutable; is valid as declared in {@link #isValidProfit(String)}
+ */
+public class ItemProfit {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Profit should only contain numbers, and it should be at most 4 digits long";
+    public static final String VALIDATION_REGEX = "^([1-9]{0,4})$";
```

Comment:
For `ItemProfit`, we are using the same `VALIDATION_REGEX`. However when we calculate profit:
```
Double a = 5.00;
Double b = 1.00;
Double profit = a - b; // profit will be 4.0, NOT 4.00 (INVALID regex)
```

Reply:
Thank you I will implement after this PR merges